contrib/typst/book/lib.typ

```diff
@@ -135,16 +135,23 @@
 
 // Collect text content of element recursively into a single string
 // https://discord.com/channels/1054443721975922748/1088371919725793360/1138586827708702810
+// https://github.com/Myriad-Dreamin/typst-book/issues/55
 #let plain-text(it) = {
   if type(it) == str {
-    it
+    return it
   } else if it == [ ] {
-    " "
-  } else if it.func() == text or it.func() == raw {
+    return " "
+  }
+  let f = it.func()
+  if f == smallcaps("").func() {
```

Comment:
Could we get `smallcaps("").func()` and store it into a variable for better readability?

```typ
#let _styled = smallcaps("").func();
#let _equation = $1$.func();
#let _sequence = [].func();
```

Reply:
Done